src/main/frontend/components/publishing.cljs

```diff
@@ -0,0 +1,140 @@
+            [frontend.handler.project :as project-handler]))
+
+(defn update-state-and-notify
+  [page-name]
+  (page-handler/page-add-properties! page-name {:published false})
+  (notification/show! (util/format "Remove Page \"%s\" from Logseq server success" page-name) :success))
+
+
+(defn project
+  [editor-state current-project pages]
+  (rum/with-context [[t] i18n/*tongue-context*]
+    (if (= :display @editor-state)
+      (when current-project
+        [:div.cp__publishing-pj
+         [:span.cp__publishing-pj-name current-project]
+         [:span.cp__publishing-edit
+          {:on-click
+           (fn [_]
+             (reset! editor-state :editor))}
+          (t :publishing/edit)]])
+      [:div.flex.cp__publishing_pj_edit
+       [:input#cp__publishing-project-input
+        {:placeholder current-project
+         :default-value current-project}]
+       [:div.cp__publishing-pj-bt
+        (ui/button
+          (t :publishing/save)
+          :on-click (fn [e]
+                      (util/stop e)
+                      (let [editor (.getElementById js/document "cp__publishing-project-input")
+                            v (.-value editor)
+                            data {:name v}]
+                        (-> (p/let [result (project-handler/update-project current-project data)]
+                              (when (:result result)
+                                (state/update-current-project :name v)
+                                (notification/show! "Updated project name successfully." :success)
+                                (reset! editor-state :display)))
+                            (p/catch
+                              (fn [error]
+                                (notification/show! "Failed to update project name." :failed))))))
+          :background "green")]
+
+       [:div.cp__publishing-pj-bt
+        (ui/button
+          (t :publishing/delete)
+          :on-click (fn [e]
+                      (util/stop e)
+                      (let [confirm-message
+                            (util/format
+                              "This operation will delete all the published pages under the project \"%s\", are you sure?"
+                              current-project)]
+                        (when (.confirm js/window confirm-message)
+                          (p/let [result (project-handler/delete-project current-project)]
+                            (when (:result result)
+                              (reset! editor-state :display)
+                              (doseq [page pages]
+                                (let [page (first page)
+                                      page-name (:page/name page)]
+                                  (page-handler/page-add-properties! page-name {:published false})))
+                              (state/remove-current-project)
+                              (notification/show! "The project was deleted successfully." :success))))))
+          :background "pink")]
```

Comment:
I didn't know the reason why 'red' didn't work.

Reply:
i have tested . it works ...
```clojure
(ui/button "button" :background "red")
```